x-pack/plugins/ml/server/routes/job_audit_messages.ts

```diff
@@ -96,4 +97,40 @@ export function jobAuditMessagesRoutes({ router, routeGuard }: RouteInitializati
       }
     )
   );
+
+  /**
+   * @apiGroup Annotations
```

Comment:
Shouldn't this `apiGroup` be `JobAuditMessages`?

Reply:
Yep - thanks! Updated in 9998eb65ac3e95f8c8d6d6123330e2f9dd23b45f